src/main/java/org/opensrp/service/ClientService.java

```diff
@@ -60,8 +60,9 @@ public Client getByBaseEntityId(String baseEntityId) {
 		return allClients.findByRelationshipIdAndDateCreated(relationalId, dateFrom, dateTo);
 	}
 
-	@PreAuthorize("hasRole('CLIENT_VIEW')")
-	@PostFilter("hasPermission(filterObject, 'CLIENT_VIEW')")
+	/*
+	No ACL to allow for global searching
```

Comment:
can we add more details on why this is being excluded, also the exclusion should be via a new method